app/assets/stylesheets/partials/_currency_input.scss

```diff
@@ -0,0 +1,9 @@
+
+span.prefix {
```

Comment:
please put this into separate scope like `.input-currency` - would not apply accidentally

Reply:
You mean change `span.prefix` to `span.input-currency`?   In fact, on reflection, I don't know why I didn't make the style names the same.   Any they aren't really limited to currency's - could be any prefix.

I'' have another look at the class names.

Reply:
Renamed the new css classes to be more specific to the role they are being used for.